drivers/cmd/kubernetes/main.go

```diff
@@ -78,7 +78,8 @@ func deleteJob(m *dipper.Message) {
 	client := k8client.BatchV1().Jobs(nameSpace)
 	ctx, cancel := context.WithTimeout(context.Background(), driver.APITimeout*time.Second)
 	defer cancel()
-	err := client.Delete(ctx, jobName, metav1.DeleteOptions{})
+	deletePropagation := metav1.DeletePropagationForeground
```

Comment:
I think `DeletePropagationBackground` is probably better.

https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#DeletionPropagation